package.json

```diff
@@ -3,6 +3,7 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "@craco/craco": "^6.4.5",
```

Comment:
Lol, how did this get in here?

Reply:
my fault......it wasn't working npm install

Reply:
how I can fix that?

Reply:
thank you